README.md

```diff
@@ -11,6 +11,7 @@ Go bindings for the [HuggingFace Tokenizers](https://github.com/huggingface/toke
 Build your Go application using pre-built native binaries: `docker build --platform=linux/amd64 -f example/Dockerfile .`
 
 Available binaries:
+
```

Comment:
Everything in this file is just markdown warning fixes.  Can revert if it's not wanted

Reply:
that's fine